app/javascript/src/components/Dashboard/Contacts/DeleteAlert.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+
+import { Alert, Toastr } from "neetoui";
+
+import { customizeValuesByDeletionType } from "./utils";
+
+const DeleteAlert = ({
+  setContacts,
+  onClose,
+  isOpen,
+  selectedContact,
+  setSelectedContact,
+  selectedContactIds,
+  setSelectedContactIds,
+  deletionType,
+}) => {
+  const [deleting, setDeleting] = useState(false);
```

Comment:
Correct the naming here: 
- https://academy.bigbinary.com/learn-basic-javascript/variables/boolean
- https://academy.bigbinary.com/learn-react/react-best-practices/naming-conventions-in-react#:~:text=Naming%20boolean%20variable%20and%20function

Reply:
I have corrected the issues and raised a separate PR.